server/src/main/java/io/openbytes/odi/domain/storage/Storage.java

```diff
@@ -0,0 +1,16 @@
+package io.openbytes.odi.domain.storage;
+
+import io.openbytes.odi.domain.ODIResource;
+
+import java.util.List;
+
+public interface Storage {
+    /**
+     * list odi resources
+     * @param prefix prefix you want to query, eg: abc/
+     * @param marker
+     * @param maxKeys
+     * @return todo we should update resp, add isTruncated info...
+     */
+    List<ODIResource> ListObjects(String prefix, String marker, int maxKeys);
```

Comment:
Is Objects are ODIResources or not?
if yes, should use a unified name.

Reply:
Will update to File instead.